piyushgarg-dev/review-app
# fix: added cursor pointer to theme toggle dropdown items

## What does this PR do?

Makes the cursor property as pointer for the theme toggle dropdown items, since they are clickable

Fixes #146

## Video Demo
https://www.loom.com/share/b98640d571c2435090da6aeb864e6f79



## Type of change




## How should this be tested?

1. Login or signup and go to dashboard.
2. On the upper righthand corner, click on the theme toggle button
3. Hover on the dropdown items of theme toggle button
4. You will see that the cursor is set to pointer
